2023-03-06 14:25:10,301 DEBUG    root            Loaded Command Group: ['gcloud', 'auth']
2023-03-06 14:25:10,302 DEBUG    root            Loaded Command Group: ['gcloud', 'auth', 'list']
2023-03-06 14:25:10,304 DEBUG    root            Running [gcloud.auth.list] with arguments: [--filter: "status=ACTIVE", --format: "value(account)", --quiet: "True", --verbosity: "warning"]
2023-03-06 14:25:10,306 DEBUG    root            Chosen display Format:    table[title='Credentialed Accounts'](
        status.yesno(yes='*', no=''):label=ACTIVE,
        account
    ) value(account)
2023-03-06 14:25:10,307 INFO     root            Display format: "    table[title='Credentialed Accounts'](
        status.yesno(yes='*', no=''):label=ACTIVE,
        account
    ) value(account)"
2023-03-06 14:25:10,307 INFO     ___FILE_ONLY___ wongshennan@gmail.com

2023-03-06 14:25:10,308 DEBUG    root            [Errno 32] Broken pipe
Traceback (most recent call last):
  File "/Users/wongshennan/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/Users/wongshennan/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 808, in Run
    resources = display.Displayer(command_instance, args, resources,
  File "/Users/wongshennan/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/calliope/display.py", line 491, in Display
    log.out.flush()
  File "/Users/wongshennan/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/core/log.py", line 247, in flush
    self.__stream_wrapper.stream.flush()
BrokenPipeError: [Errno 32] Broken pipe
